simple-todo/webpack.config.js

```diff
@@ -9,33 +15,29 @@ module.exports = {
     filename: 'build.js'
   },
   module: {
-    loaders: [
-      {
-        test: /\.vue$/,
-        loader: 'vue'
-      },
-      {
-        test: /\.js$/,
-        loader: 'babel',
-        exclude: /node_modules/
-      },
+    rules: [
+      { test: /\.vue$/, loader: 'vue-loader' },
```

Comment:
vue-loader can no longer be referenced as just vue.
`npm run dev` throws an error if this is missing.